ocs_ci/ocs/platform_nodes.py

```diff
@@ -1712,6 +1713,102 @@ def read_default_config(self, default_config_path):
         return default_config_dict
 
 
+class IBMPowerNodes(NodesBase):
+    """
+    IBM Power Nodes class
+    """
+
+    def __init__(self):
+        super(IBMPowerNodes, self).__init__()
+        self.powernodes = powernodes.POWERNODES()
+
+    def stop_nodes(self, nodes, force=True):
+        """
+        Stop PowerNode
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            force (bool): True for force nodes stop, False otherwise
+
+        """
+        if self.powernodes.iskvm():
+            self.powernodes.stop_powernodes_machines(nodes, force=force)
+        else:
+            raise NotImplementedError(
+                "This is not libvirt environment. Stop nodes not implemented"
+            )
+
+    def start_nodes(self, nodes, force=True):
+        """
+        Start PowerNode
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): Wait for node status
+
+        """
+        if self.powernodes.iskvm():
+            self.powernodes.start_powernodes_machines(nodes, force=force)
+        else:
+            raise NotImplementedError(
+                "This is not libvirt environment. Start nodes not implemented"
+            )
+
+    def restart_nodes(self, nodes, timeout=540, wait=True, force=True):
```

Comment:
`timeout` and `wait` args are not used or included in the docstring.

Reply:
Completed with latest push.